GCTC-NTGC/gc-digital-talent
# Feature - 3158 - Admin View Pool Page

Resolves #3158 

## Summary

This adds the new pool advertisement page to admin behind the direct intake feature flag.

## Testing

1. Ensure you enable the `FEATURE_DIRECTINTAKE` flag in `.apache_env`
2. Rebuild the php container
  1. `docker-compose stop php`
  2. `docker-compose rm php`
  3. docker-compose up --detach --build`
3. Navigate to `/admin/pools`
4. Click on the name of the pool
5. Ensure the page matches the [prototype](https://www.figma.com/proto/b6lDM86QCJQ6iaAh7qrTYH/Admin-v4-(Direct-Intake)?page-id=0%3A1&node-id=1919%3A3386&viewport=-3442%2C1747%2C0.59&scaling=min-zoom&starting-point-node-id=1618%3A6064)

## Note

The URL field just displays the title until we have on available.

## Screenshot
![localhost_8000_en_admin_pools_081a27ce-8dab-4e4c-814b-003757cd60e9](https://user-images.githubusercontent.com/4127998/177619007-9324f808-9ae1-4712-8054-0f09dcd3ab17.png)
